spikesorters/waveclus/waveclus.py

```diff
@@ -41,6 +41,14 @@ class WaveClusSorter(BaseSorter):
         'min_clus': 20,
         'maxtemp': 0.251,
         'template_sdnum': 3,
+        'enable_detect_filter': False,
```

Comment:
Hi @ferchaure 

I would leave the filters on by default. This is the same as other sorters

Reply:
Sorry, for that missing parameter.

If you leave the filters on by default you won't be able to separate the pre-processing (using spikeinterface) from the automatic one on the sorters. By the way, does Mountainsort automatically filter the input signal? I know that, for example, kilosort does it and I guess that it's quite messy to code around that differences.

If the filters will be enabled by default:
Is it possible to display a message, if the user already filtered the raw signal, to remind the user that Wave_clus will apply more filters? Especially if the sorting filter is enabled because the extra filtering can affect a lot the results in a single-channel recording.

Reply:
Hi @ferchaure

Yes it's possible. Every recording extractor now has info about whether it's filtered or not. So you can just check it with: `recording.is_filtered`.

I agree this is a good behavior :)

Reply:
cool, easy to do. Do you have a proper way to display messages to the users (with colours or something like that) ? or just a print() will be fine?

Reply:
Sorry @ferchaure, forgot to reply to this. Just a print is fine!

Reply:
I'll actually merge this and apply the changes